src/components/body/AboutMe/styles.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+
+export const H2 = styled.h2`
```

Comment:
H2 is not a good styled component name for something that is H2.
im sure you can think of a more better name